run.py

```diff
@@ -61,6 +61,8 @@
         [--xunit-results]
   run.py --cleanup=name [--osp-cred <file>]
         [--log-level <LEVEL>]
+  run.py --clean-volumes=yes [--osp-cred <file>]
```

Comment:
IMO, the clean up of orphaned volumes can be a separate script. This script uses a generic scope to perform action whereas `run` uses a restrictive scope. That is, either it is performed against a `pattern` or `reuse`.

Reply:
Add option to run.py to keep single script to run different options.

Reply:
We can enhance to restrict to run with specific pattern or user-id. Also as a prevention Openstack does not delete in-use volumes, so risk of deleting unintended vols is minimal.